.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I don't think we should have a .vscode in here

Reply:
This change has been made accordingly